sdk/scheduler/src/main/java/com/mesosphere/sdk/config/DefaultConfigurationUpdater.java

```diff
@@ -117,9 +118,14 @@ public UpdateResult updateConfiguration(ServiceSpec candidateConfig) throws Conf
                                 "%d Errors: %s", errors.size(), sj.toString()));
             }
         } else if (targetConfig == null || !configComparator.equals(targetConfig, candidateConfig)) {
-            LOGGER.info("Changes detected between current target configuration '{}' and new " +
-                            "configuration. Setting target to new configuration.",
-                    targetConfigId);
+            if (targetConfig == null) {
```

Comment:
If the scheduler gets restarted while still in the middle of an initial deployment, will this result in the remainder of that deployment appearing to be an 'update'?